CareKitStore/CareKitStoreTests/CoreDataSchema/TestCoreDataSchema+Migrations.swift

```diff
@@ -48,6 +48,7 @@ class TestCoreDataSchemaMigrations: XCTestCase {
     ///   Tasks: 3
     ///   Outcomes: 3
     ///   OutcomeValues: 3
+    /* If current CareKitStore is CareKitStore2.1
```

Comment:
These lines are commented out, are you planning to do something or are planning to remove?